src/textual/renderables/bar.py

```diff
@@ -130,7 +142,7 @@ def frange(start, end, step):
 
     from rich.live import Live
 
-    bar = Bar(highlight_range=(0, 4.5), width=80)
+    bar = Bar(highlight_range=(0, 4.5), width=10, thickness=2)
```

Comment:
Reset the demo example.

```suggestion
    bar = Bar(highlight_range=(0, 4.5), width=80, thickness=1)
```

Reply:
Fixed.